packages/react-composites/src/composites/common/AddPeopleDropdown.tsx

```diff
@@ -69,14 +73,17 @@ export const AddPeopleDropdown = (props: AddPeopleDropdownProps): JSX.Element =>
       });
     }
 
-    menuProps.items.push({
-      key: 'DialpadKey',
-      text: strings.openDialpadButtonLabel,
-      itemProps: { styles: copyLinkButtonStylesThemed },
-      iconProps: { iconName: PeoplePaneOpenDialpadIconNameTrampoline(), style: iconStyles },
-      onClick: () => setShowDialpad(true),
-      'data-ui-id': 'call-with-chat-composite-dial-phone-number-button'
-    });
+    // only show the dialpad option when alternateCallerId is set
+    if (alternateCallerId) {
+      menuProps.items.push({
+        key: 'DialpadKey',
+        text: strings.openDialpadButtonLabel,
+        itemProps: { styles: copyLinkButtonStylesThemed },
+        iconProps: { iconName: PeoplePaneOpenDialpadIconNameTrampoline(), style: iconStyles },
+        onClick: () => setShowDialpad(true),
+        'data-ui-id': 'call-with-chat-composite-dial-phone-number-button'
+      });
+    }
 
     return menuProps;
   }, [
```

Comment:
You need to add `alternateCalledId` here as well

Reply:
This is the menu option to open the dialpad, so it doesn't need alternatecallerid. alternateCalledId is passed to the callingdialpad this menu option opens